report/test/renderer/dom-test.js

```diff
@@ -19,23 +19,22 @@ describe('DOM', () => {
   /** @type {DOM} */
   let dom;
   let window;
+  let nativeCreateObjectURL;
 
   beforeAll(() => {
     Util.i18n = new I18n('en', {...Util.UIStrings});
     window = new jsdom.JSDOM().window;
 
-    // Make a lame "polyfill" since JSDOM doesn't have createObjectURL: https://github.com/jsdom/jsdom/issues/1721
-    const lameCOURL = jest.fn(_ => `https://fake-origin/blahblah-blobid`);
-    if (!URL.createObjectURL) {
-      URL.createObjectURL = lameCOURL;
-    }
+    nativeCreateObjectURL = URL.createObjectURL;
```

Comment:
add a comment that we're shimming because the Node version of `URL.createObjectURL` isn't compatible with the jsdom blob type, at least at this time?